plenty_api/api.py

```diff
@@ -1235,6 +1237,31 @@ def plenty_api_create_booking(self, order_id: int,
                                              data=data)
         return response
 
+
+    def plenty_api_create_property_selection_name(self, json: dict) -> dict:
+        """
+        Create a language for a selection property.
+
+        Parameter:
+            json            [dict]  -   single Json object describing the
+                                        creation
```

Comment:
As mentioned above please replace this JSON with the 3 values directly as parameters.